site/getting-started.md

```diff
@@ -79,7 +79,7 @@ If you encounter issues, review the [troubleshooting docs][5], [file an issue][6
 [1]: /docs/v1.0.0/deploy-options
 [2]: /docs/v1.0.0/httpproxy
 [3]: /docs
-[4]: {{ site.github.repository_url }}/tree/master/FAQ.md
+[4]: {% link _resources/faq.md %}
```

Comment:
@davecheney is suggesting that we prefer `{% link %}` because that ensures broken links are found when Jekyll starts up, rather than in the post-build link checks. That' s probably less of an issue once we have link validation in CI.

Reply:
I'd like to defer the dicussion on the right link format to use to issue #1891 so it does not delay getting these links fixed.

I have added a reply to the issue detailing the differences between the two approaches.